### DISCUSSION TITLE
Append to list

### DISCUSSION THREAD

**Discussion by napter** [sensedeep/dynamodb-onetable#507] [Q&A]
How do you append an item to a list (Array type)? In dynamodbtoolbox you can set the value to { $append: [...] }. I couldn't find anywhere in the docs explaining how to do it in this library.

**Comment by mobsense** [comment]
Here is a quick example that uses `set` with `list_append`

```
    let user = await User.create({
        email: "user@example.com",
        addresses: ['14 Park Lane']
    }, {log: true})

    user = await User.update({
        email: user.email,
    }, {
        log: true,
        set: {addresses: 'list_append(addresses, @{newAddress})'},
        substitutions: {
            newAddress: ['25 Mayfair']
        }
    })
```

**Comment by mobsense** [comment]
Updated the doc at:

https://doc.onetable.io/api/model/set/
